compat/server.js

```diff
@@ -1,5 +1,13 @@
 /* eslint-disable */
-var renderToString = dep(require('preact-render-to-string'));
+var renderToString
+try {
+	renderToString = dep(require('preact-render-to-string'));
+} catch (e) {
+	throw new Error(
+		'You seem to be missing the "preact-render-to-string" dependency.\n' +
+		'You can add this by using "npm install --save preact-render-to-string".'
```

Comment:
Maybe we should point to `preact-render-to-string@next` for now. The `latest` tag only works with Preact 8.x